kikuchipy/signals/ebsd_master_pattern.py

```diff
@@ -61,6 +74,145 @@ def __init__(self, *args, **kwargs):
         self.projection = kwargs.pop("projection", None)
         self.hemisphere = kwargs.pop("hemisphere", None)
 
+    def get_patterns(
+        self,
+        rotations: Rotation,
+        detector: EBSDDetector,
+        energy: int,
+        n_chunk: Optional[int] = -1,
+        dtype_out: Optional[np.dtype] = np.float32,
+        compute: Optional[bool] = False,
+    ) -> Union[EBSD, LazyEBSD]:
+        """Creates a dictionary of EBSD patterns for a sample in the
+        (RD, TD, ND) reference frame, given a set of local crystal lattice
```

Comment:
Great that you mention the reference frame here, I think "in the EDAX TSL (RD, TD, ND) reference frame, ..." would be even clearer.

We can decide in a few weeks, when we have tested this functionallity more, whether we should rotate the orientations internally to the Bruker reference frame, or just rotate to the scan reference frame.